cts/lab/patterns.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, name):
         ]
         
         self.components["corosync-ignore"] = [
-            r"error:.*Connection to the CPG API failed: Library error",
+            r"Could not connect to Corosync CFG: CS_ERR_LIBRARY",
```

Comment:
As I found in #2535, you're going to need to ignore another new log message that shows up in the CTSlab ResourceActivity output.  I've added the following patch to my repo for that PR, but it should probably go somewhere else, and would be more appropriate in this PR in general.
```
commit 00a278f15b7ae0b29894a043badf5ef6ec605f20 (HEAD -> schedulerd)
Author: Chris Lumens <clumens@redhat.com>
Date:   Mon Nov 1 11:51:15 2021 -0400

    Test: CTS: Ignore some ResourceActivity log messages.
    
    9a197c7 introduced a new log message that will be printed out during the
    Reattach test, and possibly others.  This message can be printed out
    even if nothing's wrong, which will still result in it being in the log,
    which will make the test fail.  Ignore it if everything's okay.

diff --git a/cts/lab/CTStests.py b/cts/lab/CTStests.py
index 649489cf6..d65f9939b 100644
--- a/cts/lab/CTStests.py
+++ b/cts/lab/CTStests.py
@@ -1735,7 +1735,7 @@ class Reattach(CTSTest):
             self._set_managed(node)
             return self.failure("Couldn't restart the cluster")
 
-        if self.local_badnews("ResourceActivity:", watch):
+        if self.local_badnews("ResourceActivity:", watch, [r"Result of .* operation for .* on .*: ok"]):
             self._set_managed(node)
             return self.failure("Resources stopped or started during cluster restart")
```

Reply:
Nope, any start/stop operations indicate a problem. Not necessarily a problem that anything needs to be done about, but it's not good :)

There is one known scenario that can be ignored. If a previous ComponentFail test killed pacemaker-execd, and led to "connected" transient node attribute being unset, that will stop the stateful-1 resource since it has a constraint tied to that attribute. When Reattach restarts the cluster, the probes will re-set the node attribute, thus allowing stateful-1 to come back up again. This could be considered a problem with the initial ComponentFail recovery; if a daemon respawn might have caused transient attributes to go away, then perhaps we should reprobe everything in case probes set attributes, but that's not a perfect solution, and the current behavior could also be seen as a legitimate recovery choice.

All bets are off when running under valgrind -- that causes numerous spurious errors. Valgrind should only be used for the valgrind output, not to judge the lab tests themselves.